cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -49,8 +49,24 @@ akka.management {
       stable-margin = 5 seconds
 
       # Interval at which service discovery will be polled in search for new contact-points
+      #
+      # Note that actual timing of lookups will be the following:
+      # - perform initial lookup; interval is this base interval
+      # - await response within resolve-timeout
+      #   (this can be larger than interval, which means interval effectively is resolveTimeout + interval,
+      #    this has been specifically made so, to not hit discovery services with requests while the lookup is being serviced)
+      #   - if failure happens apply backoff to interval (grow interval by multiplying by the exponential-backoff-factor)
+      # - if no failure happened, and we receive a resolved list of services, schedule another lookup in interval time
+      #   - if previously failures happened during discovery, a successful lookup resets the interval to `interval` again
+      # = repeat until stable-margin is reached
       interval = 1 second
 
+      # When lookups in discovery fail an exponential backoff will be performed
+      exponential-backoff-factor = 1.2
```

Comment:
I don't think this property is needed. It's too detailed for anyone to care. Using pow 2 with a max cap is enough. That's how we do it in other backoff features, such as `RestartFlow` and `BackoffSupervisor`. The exact calculation is in https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala#L210

Reply:
Thanks, will do that then :)